ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -82,30 +99,38 @@ public void onAnimationEnd(final Animator animation) {
         }
     };
 
+    //region Initialization
     public PageContentLayout(@NonNull final Context context) {
         this(context, null);
+        initialization(context);
     }
 
     public PageContentLayout(@NonNull final Context context, @Nullable final AttributeSet attrs) {
         this(context, attrs, 0);
+        initialization(context);
```

Comment:
`initialization()` would have already been called at this point.

Reply:
So these aren't overloaded Constructors?

Reply:
they are calling `this` and not `super`